### ISSUE TITLE
Docker stop integration test(s) failing with 0.5 tag build against VC

### ISSUE THREAD

**Issue by corrieb** [vmware/vic#1980]
Using build 3991, I came across a bunch of failures in the docker stop tests. 
```
$ pybot tests/test-cases/Group1-Docker-Commands/1-7-Docker-Stop.robot
==============================================================================
1-7-Docker-Stop :: Test 1-7 - Docker Stop                                     
==============================================================================
10.118.67.35

Installing VCH to test server...
Installer completed successfully: VCH-0-6682...
Stop an already stopped container                                     | PASS |
------------------------------------------------------------------------------
Basic docker container stop                                           | FAIL |
1 != 0
------------------------------------------------------------------------------
Stop a container with SIGKILL using default grace period              | FAIL |
1 != 0
------------------------------------------------------------------------------
Stop a container with SIGKILL using specific stop signal              | FAIL |
1 != 0
------------------------------------------------------------------------------
[ WARN ] Issue #1924 is blocking implementation
Stop a container with SIGKILL using specific grace period             | PASS |
------------------------------------------------------------------------------
Stop a non-existent container                                         | PASS |
------------------------------------------------------------------------------
Attempt to stop a container that has been started out of band         | FAIL |
1 != 0
------------------------------------------------------------------------------
```

**Comment by corrieb** [comment]
Seems like the step where the debug is downloaded is failing

[suite-html.zip](https://github.com/vmware/vic/files/418850/suite-html.zip)
[VCH-0-6682-container-logs.zip](https://github.com/vmware/vic/files/418849/VCH-0-6682-container-logs.zip)

**Comment by mdubya66** [comment]
While this is on VC and not part of our integration tests yet it is a show stopper. Assigning to @hickeng for initial triage

**Comment by mhagen-vmware** [comment]
Fixed with #2474
